src/main/java/uk/nhs/digital/nhsconnect/lab/results/fixtures/EdifactFixtures.java

```diff
@@ -0,0 +1,99 @@
+package uk.nhs.digital.nhsconnect.lab.results.fixtures;
+
+import java.util.List;
+
+public class EdifactFixtures {
```

Comment:
This should be in the `test` folder, not `main`

Reply:
Done in b4a4923